tools/cldr-apps/src/main/java/org/unicode/cldr/web/api/LocaleCompletion.java

```diff
@@ -142,25 +142,34 @@ static LocaleCompletionResponse getLocaleCompletion(final CLDRLocale cldrLocale,
 
         final List<CheckStatus> results = new ArrayList<>();
         for (final String xpath : file) {
+            lcr.allXpaths ++;
```

Comment:
unusual space before "++" 

i wish we were using similar formatting tools for java as we're using for js